config/ModulesMapping.jsonc

```diff
@@ -1,92 +1,92 @@
 {
-    "Analytics": "^activitystatistics\\.|^users.userAnalytics$|^users.officeGraphInsights$",
-    "Bookings": "^bookingBusinesses\\.|^bookingCurrencies\\.",
-    "DevicesApps.MobileAppManagement": "^users.managedAppRegistration$|^users.windowsInformationProtectionDeviceRegistration$",
-    "DevicesApps.OfficeConfiguration": "^officeConfiguration\\.",
-    "DevicesApps.SharedResources": "^users.mobileAppTroubleshootingEvent$",
-    "Education": "^education\\.",
-    "Files.Drives": "^drives\\.|^drive\\.",
-    "Files.Permissions": "^groups.resourceSpecificPermissionGrant$", // Missing in docs
-    "Files.Shares": "^shares\\.",
-    "Groups.Calendar": "^groups.calendar$|^groups.event$",
-    "Groups.Conversation": "^groups.conversation$",
-    "Groups.ConversationThread": "^groups.conversationThread$",
+    // "Analytics": "^activitystatistics\\.|^users.userAnalytics$|^users.officeGraphInsights$",
```

Comment:
Why are these being commented out?

Reply:
https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/88 has to be resolved first before this is merged.